Makefile

```diff
@@ -3,6 +3,7 @@ BINPATH ?= build
 BUILD_TIME=$(shell date +%s)
 GIT_COMMIT=$(shell git rev-parse HEAD)
 VERSION ?= $(shell git tag --points-at HEAD | grep ^v | head -n 1)
+CORE_ASSETS_PATH=$(shell go list -f '{{.Dir}}' -m github.com/rav-pradhan/test-modules/render)
```

Comment:
We set this variable here to the specific cached version of the renderer library being used by the app